apps/Minecraft Pi (Modded)/install

```diff
@@ -1,37 +1,36 @@
 #!/bin/bash
-"${DIRECTORY}/api" apt_lock_wait
 
-# Remove old MCPILs
-sudo apt-get remove -y mcpil-r mcpil &>/dev/null || true
+# Version
+version="196/artifact/out/minecraft-pi-reborn-client-2.3.5"
 
-# Install the repos
-wget -qO- https://raw.githubusercontent.com/MCPI-Revival/mcpi-packages/master/install.sh | grep -v 'sudo apt update' | sudo bash - || error "Failed to install APT repo!"
+# Install
+case "${arch}" in
+    "64") do url_arch=armhf ;;
+    "32") do url_arch=armhf ;;
```

Comment:
@TheBrokenRail, can you confirm that you really intended to have armhf in both places?

Reply:
Fixed!